lib/agent0/agent0/hyperdrive/policies/deterministic.py

```diff
@@ -0,0 +1,111 @@
+"""Deterministically trade things."""
+# pylint: disable=line-too-long, missing-class-docstring
```

Comment:
why disable these?

Reply:
good catch, they're from an old version where this was like 5 (very long) lines

Reply:
good catch, they're from an old version where this was like 5 (very long) lines